src/tests/hash_functions/Makefile.manual

```diff
@@ -1,3 +1,51 @@
-PROGS_SRC = test_64_bit_hash_performance.f90 test_32_bit_hash_performance.f90
+CC = gcc
+CXX = g++
+
+FFLAGS = -Wall -Wextra -Wimplicit-interface -fPIC -g -fcheck=all -fno-range-check
+CFLAGS = -O3
+CXXFLAGS = -O3
+
+CPPFLAGS += -I. -I../.. -I..
+LDFLAGS += -L../.. -L.. -lstdlib-testing -lstdlib
+
+PROGS =  test_hash_functions
+TESTPROGS = $(PROGS:=TEST)
+
+all: $(PROGS)
+
+test: $(TESTPROGS)
+
+$(TESTPROGS):
+	./$(@:TEST=)
+
+test_hash_functions: test_hash_functions.f90 generate_hash_arrays.o libc_hash.a
+	$(FC) $(FFLAGS) $(CPPFLAGS) -L. -o $@ $^ $(LDFLAGS) -lc_hash -lstdc++
```

Comment:
This looks as if you are still going to write the hash files and then read them, while they could be generated and used without the intermediate transfers to and from a binary file.

Reply:
This is indeed the case. My skills in C/C++ are not that high, and I didn't want spend time in something that might not work (especially for Intel OneAPi). If we can get it work, then I think it will be worthwhile that we spend time to improve this approach.